org.zenframework.z8.compiler/src/main/java/org/zenframework/z8/compiler/core/IAttribute.java

```diff
@@ -14,6 +14,8 @@
 	final static String Activator = "activator";
 	final static String Entry = "entry";
 	final static String Job = "job";
+	final static String ApiDescription = "apiDescription";
+	final static String СontentParams = "contentParams";

```

Comment:
Rename "contentParams" to "apiActions". [apiActions] describes any action in JSON format:
{
    "read": "Read params...",
    "content": "Content params..."
}